include/regions.h

```diff
@@ -0,0 +1,44 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef __LABWC_REGIONS_H
+#define __LABWC_REGIONS_H
+
+struct seat;
+struct view;
+struct server;
+struct output;
+struct wl_list;
+struct wlr_box;
+struct multi_rect;
+
+/* Double use: rcxml.c for config and output.c for usage */
+struct region {
+	struct wl_list link; /* struct rcxml.regions, struct output.regions */
+	char *id;
+	struct wlr_box geo;
+	struct wlr_box percentage;
+	struct {
+		int x;
+		int y;
+	} center;
+};
+
+struct region_overlay {
+	struct wlr_scene_tree *tree;
+	union {
+		struct wlr_scene_rect *overlay;
+		struct multi_rect *pixman_overlay;
+	};
+};
+
+void regions_reconfigure(struct server *server);
+void regions_reconfigure_output(struct output *output);
+void regions_update_geometry(struct output *output);
+void regions_evacuate_output(struct output *output);
```

Comment:
Consider adding a comment here do describe what `regions_evacuate_output()` does.

Reply:
Added a comment, I hope its somewhat understandable for somebody not inherently used to the codebase.